frontend/src/old-pages/Clusters/__tests__/util.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { getScripts } from "../util";
+
+describe("given custom actions", () => {
+    describe("when scripts are present", () => {
+        it("should return the script names", () => {
```

Comment:
suggestion: this doesn't fully capture what the function does. (I know I sent you these lines, let's find something that can document the function)

Here's what I would do:

1. split into two different `it` tests, one per supported custom action. Meaning, one for the `OnNodeStart` and one for the `OnNodeConfigured` since those are the only ones we actually support
2. explicit what this means in terms of business logic, rather than implementation. For instance, you could write `given a custom action to be run before any node deployment, when a custom script is provided, it should return the name of the script`, same goes for the `OnNodeConfigured`
3. test corner cases: `when no custom script is provided` or `when the script should be run with args`